### ISSUE TITLE
Add results for dataset BungeeNeRF

### ISSUE THREAD

**Issue by ZhiyeTang** [w-m/3dgs-compression-survey#13]
The [BungeeNeRF dataset](https://github.com/city-super/BungeeNeRF?tab=readme-ov-file) is also a representative real-world, large-scale dataset for the NVS task. Many works in 3DGS compression have been evaluated on the BungeeNeRF dataset. Adding this dataset to the project will better reflect the performance of different methods.

**Comment by w-m** [comment]
Fully agreed that adding a large-scale dataset would help make the comparisons more interesting. But finding a dataset that is supported by a majority of methods is difficult. If only a handful of methods provide number for a dataset, it does not make for a valuable comparison. That's why we settled on using the datasets used in original 3DGS, as most methods compare against them.

How many methods have numbers for BungeeNeRF? Could you list the methods that provide BungeeNeRF evaluation data in their paper, or in their repo?

**Comment by ZhiyeTang** [comment]
Agreed. To the best of my knowledge, HAC has reported experimental results of some listed approaches on BungeeNeRF dataset, as follows:
![image](https://github.com/user-attachments/assets/b4266a8a-7d5f-452b-b9be-90e0f7c55508)

Additionally, I've evaluated CompGS (ACMMM'24) and reduced-3DGS (I3D'24) on BungeeNeRF. Is this helpful?
